.github/workflows/ci.yml

```diff
@@ -111,7 +112,7 @@ jobs:
       if: matrix.platform.name != 'Android' && matrix.config.name != 'Frameworks' && matrix.config.name != 'iOS' && matrix.config.name != 'Static DRM'
       shell: bash
       run: |
-        cmake -S $GITHUB_WORKSPACE/test/install -B $GITHUB_WORKSPACE/test/install/build -DSFML_ROOT=$GITHUB_WORKSPACE/install -DCMAKE_VERBOSE_MAKEFILE=ON ${{matrix.platform.flags}} ${{matrix.config.flags}} ${{matrix.type.flags}}
+        cmake -S $GITHUB_WORKSPACE/test/install -B $GITHUB_WORKSPACE/test/install/build -DSFML_ROOT=$GITHUB_WORKSPACE/install -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_BUILD_TYPE=${{ matrix.type.name == 'Debug' && 'Debug' || 'Release' }} ${{matrix.platform.flags}} ${{matrix.config.flags}} ${{matrix.type.flags}}
```

Comment:
Does `-DCMAKE_BUILD_TYPE=${{matrix.type.name}}` not work?

Reply:
It does and I actually already wrote a commit that exploits that fun little trick.

https://github.com/ChrisThrasher/SFML/commit/3945a15421bff93fb0ebe94ef66cb8b0dd589922

Reply:
Not all jobs define `matrix.type.name` so that would not work as is.